Rakefile

```diff
@@ -117,6 +156,48 @@ namespace :test do
 
       sh beaker
     end
+
+    desc "Do an ezbake build, and then a beaker smoke test off of that build, preserving the vmpooler host"
+    task :bakeNbeak do
+      package_version = nil
+
+      Open3.popen3("lein with-profile ezbake ezbake build 2>&1") do |stdin, stdout, stderr, thread|
+        # sleep 5
+        # puts "STDOUT IS: #{stdout}"
+        stdout.each do |line|
+          if match = line.match(%r|^Your packages will be available at http://builds.delivery.puppetlabs.net/puppetserver/(.*)$|)
+            package_version = match[1]
+          end
+          puts line
+        end
+        puts "PACKAGE VERSION IS #{package_version}"
```

Comment:
Good call, but I'm not sure when I will have time to work on this again, so maybe somebody could just hack that in after we merge, next time they run into it?